Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -1999,6 +2048,103 @@ def fetch_incidents(client: Client, last_run: Dict[str, Any], params: Dict[str,
     return incidents, ids_to_insert, updated_last_run_time
 
 
+def access_key_create_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    access_key = client.access_key_creation(args)
+    transform_access_keys = {
+        'Id': access_key.get('id'),
+        'Secret Key': access_key.get('secretKey'),
+    }
+
+    markdown_table = tableToMarkdown('PrismaCloud Access Key Creation', transform_access_keys, headers=['Id', 'Secret Key'])
+    return CommandResults(
+        outputs_prefix="PrismaCloud.AccessKeys",
+        outputs_key_field='id',
+        outputs=access_key,
+        raw_response=access_key,
+        readable_output=markdown_table,
+    )
+
+
+def get_access_keys_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    return get_access_key_by_id(client, args) if args.get('access-key') else get_access_keys_list(client, args)
+
+
+def get_access_key_by_id(client: Client, args: Dict[str, Any]) -> CommandResults:
+    access_key = client.get_access_key_by_id(args)
+    transform_access_key = {
+        'ID': access_key.get('id'),
+        'Name': access_key.get('name'),
+        'Expires On': timestamp_to_datestring(access_key.get('expiresOn'), DATE_FORMAT)
+    }
+
+    markdown_table = tableToMarkdown('PrismaCloud Access Key', transform_access_key, headers=['ID', 'Name', 'Expires On'])
+    return CommandResults(
+        outputs_prefix="PrismaCloud.AccessKeys",
+        outputs_key_field='id',
+        outputs=access_key,
+        raw_response=access_key,
+        readable_output=markdown_table,
+    )
+
+
+def get_access_keys_list(client: Client, args: Dict[str, Any]) -> CommandResults:
+    access_keys = client.get_access_keys_list()
+    limit = int(args.get('limit', ACCESS_KEYS_LIST_DEFAULT_LIMIT))
```

Comment:
Does the API support limit? or is it implemented only locally on our side? If the API supports it lets send it to the API instead getting all keys and slicing

Reply:
Unfortunately API does not support limit